api-extractor.json

```diff
@@ -45,7 +45,7 @@
    *
    * SUPPORTED TOKENS: <projectFolder>, <packageName>, <unscopedPackageName>
    */
-  "mainEntryPointFilePath": "<projectFolder>/protos/protos.d.ts",
```

Comment:
@fhinkel with your later stuff, does this get rewritten to `build/src/index.d.ts` broadly?